code/engine/spatial_grid.lua

```diff
@@ -26,6 +25,16 @@ function spatial_grid:new_client(position, dimensions, guid)
     return client
 end
 
+function spatial_grid:draw_debug()
+    for y=0, self.bounds.y_max-1, TILE_SIZE do
+        for x=0, self.bounds.x_max-1, TILE_SIZE do
+          love.graphics.rectangle("line", x, y, TILE_SIZE, TILE_SIZE)
+        end
+      end
+      love.graphics.setColor(1,0,1,1)
+      love.graphics.rectangle("line", 0, 0, self.bounds.x_max, self.bounds.y_max)
+      love.graphics.setColor(1,1,1,1)
```

Comment:
Tycker du inte om space? Ha gärna space så det blir (setColor(1, 1, 1, 1)
Samma med rad 29 och 30 for y = 0, osv